hooks/nuke_batch_render_movie.py

```diff
@@ -84,116 +84,113 @@ def render_in_nuke(path_to_frames, path_to_movie, extra_write_node_mapping, widt
     """
 
     output_node = None
-    root_node = nuke.root()
+    try:
+        root_node = nuke.root()
```

Comment:
Move everything into a try...except block so that any exception anywhere is returned to the calling process in the correct format.